templates/block/block--mainnavigation.html.twig

```diff
@@ -42,7 +42,7 @@
   {{ title_prefix -}}
   <h2{{ title_attributes.setAttribute('id', heading_id) }}>{{ configuration.label }}</h2>
   {{- title_suffix }}
-  <label class="visually-hidden" for="menu-toggle" onclick="">Click me to toggle menu</label>
+  <label class="visually-hidden toggler-label" for="menu-toggle" onclick="">{{ 'Click me to toggle menu'|t }}</label>
```

Comment:
DC - not sure why there is an empty onclick handler?

Also, "Click me" is device dependent language.  Could be touch or keypress or even a spoken command.  Prob best to just have "Toggle menu" or "Open menu" (latter may be better just in case someone doesn't know what toggle means! - such people do exist)

Reply:
Suppose it would have to switch between "Open menu" or "Show menu" and "Close menu" or "Hide menu"?

Reply:
I can honestly say I'm don't know why the empty onclick is there, so I'll remove. Can we compromise on the wording and say 'Open or close the menu'